core/src/main/java/io/questdb/cairo/mv/MatViewRefreshJob.java

```diff
@@ -191,7 +191,7 @@ private boolean insertAsSelect(
             TableWriterAPI tableWriter,
             long baseTableTxn,
             long refreshTriggeredTimestamp
-    ) throws SqlException {
```

Comment:
@bluestreak01 this exception is actually thrown by this method, so it has to be declared. The branch no longer compiles. Fixed it in c0ac4f4a9230be920b2ba1e74e7d534a9909a6d6

Reply:
It was highlighted as unused, sorry!

Reply:
No, worries. IDEA indeed highlights it - looks like a bug in the IDE.